src/cljs/harja/views/hallinta/kojelauta.cljs

```diff
@@ -90,9 +111,12 @@
          (yleiset/tila-indikaattori "hylatty" {:fmt-fn (constantly "Ei tavoitepistemäärää")})
          (yleiset/tila-indikaattori "valmis" {:fmt-fn (constantly "Ok")}))]]]))
 
+;; vuosi, jota ennen tavoite ja kattohinnan paatokset eivat olleet sidoksissa toisiinsa
+(def nayta-kattohintapaatos-kynnysvuosi? 2021)
```

Comment:
Oisko tää tonne tuck tiedot avaruuteen? Tämä ei oo myöskään boolean, ja, oisko kans defonce? 

Näitä ns staattisia muuttujia ruukattu nimetä tyyliin +kattohinta-kynnysvuosi+

Reply:
siirsin ja nimesin uudelleen, kiitos